.github/workflows/local-e2e-test.yml

```diff
@@ -23,8 +23,13 @@ jobs:
           node-version: ${{ matrix.node }}
```

Comment:
Any issue with adding `cache: yarn` to `actions/setup-node`'s config?

Reply:
I can't comment ([yet](https://togithub.com/github/roadmap/issues/347)) on the `actions/checkout` config, but should we set the depth to maybe 5? Otherwise, it pulls the entire git history 😱

Reply:
am fine with both of those changes 👍🏻